prometheus/cli.py

```diff
@@ -55,9 +56,15 @@ def metrics(host, token, interval, time, skip_namespaces, output):
 
     keys = list(json.loads(d.replace('\'', '\"'))
                 for d in combined_metrics.keys())
+    [key.update({'uniqueId': ':'.join(map(str, key.values()))})
+     for key in keys]
+    keys = list({k: v for k, v in key.items() if k == 'uniqueId'}
+                for key in keys)
+
```

Comment:
Let's move this parsing into prometheus.Prometheus.Metric

Reply:
Ok, let's hold that since I found a bug, I'll fix it

Reply:
Fixed